core/tests/revert-test/tests/revert-and-restart-en.test.ts

```diff
@@ -250,7 +250,7 @@ describe('Block reverting test', function () {
         while (true) {
             const lastCommitted: BigNumber = await main_contract.getTotalBlocksCommitted();
             console.log(`lastExecuted = ${lastExecuted}, lastCommitted = ${lastCommitted}`);
-            if (lastCommitted.sub(lastExecuted).gte(2)) {
+            if (lastCommitted.sub(lastExecuted).gte(1)) {
```

Comment:
Why?

Reply:
<img width="388" alt="image" src="https://github.com/matter-labs/zksync-era/assets/8031374/ac921798-c1db-4dc8-96c3-0593bf0177ee">

Reply:
It doesn't explain why it's needed, if you read comments then you'd see diff should be equal to 2